images/hubble-proto/Makefile.docker

```diff
@@ -6,6 +6,7 @@
 
 PROTOC ?= protoc
 
+EXTERNAL_PROTO_SOURCE := /usr/local/include
```

Comment:
this path should be auto-inlcuded, no?

Reply:
It doesn't seem to be the case when running as a regular user.